# Emergency STOP Status

uint32 EstopStatus

uint32 ESTOP_REASON_CLEAR=0
